src/types.ts

```diff
@@ -31,7 +31,13 @@ export type CollectionExporterOptions = {
 	query?: Pick<Query, 'filter'|'sort'|'limit'>;
 	prefix?: string;
 	onExport?: (item: Item, srv: ItemsService) => Promise<Item | null>;
-	onImport?: (item: Item, srv: ItemsService) => Promise<Item | null>;
+	onImport?: (item: Item, srv: ItemsService, opts?: ItemImportOptions) => Promise<Item | null>;
+}
+
+export type ItemImportOptions = {
```

Comment:
Can you update the README file with these options and a clear definition of how that can be used. Feel free to make a new .md file with a link to it, to explain the problems and how this can help solve it.